.github/workflows/nightly.yaml

```diff
@@ -0,0 +1,63 @@
+name: Nightly
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 1 * * *"
+
+env:
+  PYTHON_VERSION: 3.8
```

Comment:
Should we make this 3.9 to match HA? Guess it doesn't really matter.

Reply:
Right now it matches the same as the frontend uses to build its release 🤷 
I kept changing the built environment out of scope. 

Besides Python we should/cloud also consider replacing Node with 16 (as its faster, we could get faster builds).